test/functional/neutronless/bigip_interaction.py

```diff
@@ -197,26 +188,74 @@ def check_resulting_cfg(cls, test_name=current_test):
 
         test_name := the name of the test currently in tearDown
         """
-        cls.check_symbols()
-        cls._resulting_bigip_cfg(test_name)
+        if test_name:
+            cls.__current_test = test_name
+        else:
+            test_name = cls.__current_test
+        if not hasattr(pytest.symbols, 'no_bigip_tracking') and cls.ssh_cmd:
```

Comment:
This if-statement would be easier to read if the outer if tested for hasattr and the inner if tested for cls.ssh_cmd.